### PR TITLE
DP-2485:  pandas docu 🐼 

### PR DESCRIPTION
**Pull request by Mara3l**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-pandas/gooddata_pandas/utils.py [hunk 1]
```diff
@@ -24,15 +24,39 @@
 
 
 def _unique_local_id() -> str:
+    """
+    Generate unique local ID of a DataIem without dashes.
```
Comment (hkad98) [inline gooddata-pandas/gooddata_pandas/utils.py:28]: DataIem

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by hkad98 — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR299

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/116/ : FAILURE in 55s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/266/ : SUCCESS in 51s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/269/ : SUCCESS in 10s

**Pull request merged**